### DISCUSSION TITLE
How to check redis connect or not? is there any command to check ?

### DISCUSSION THREAD

**Discussion by himanshucodebrik** [ra1u/redis-dart#73] [Q&A]

**Comment by ra1u** [comment]
Can you tell more about your requirements.
We currently have 3 ways to connect on redis at lowest level. One is using, `connect`, other with tls using `connectSecure`, and also using any other provided Socket using` RedisConnection.connectWithSocket(Socket s)` . It is in general up to implementation to handle disconnections.
Other option to poll redis about connection at higher level is to issue [PING](https://redis.io/commands/ping/) command
